process/receiptsHolder.go

```diff
@@ -0,0 +1,8 @@
+package process
+
+import "github.com/ElrondNetwork/elrond-go-core/data/block"
+
+// ReceiptsHolder holds the content to be saved in (loaded from) the receipts unit
+type ReceiptsHolder struct {
+	Miniblocks []*block.MiniBlock
+}
```

Comment:
why not create here a proper exported constructor instead of the private one?

Reply:
Added proper constructor.